solutions/USACO open18-familytree.mdx

```diff
@@ -0,0 +1,127 @@
+---
+id: usaco-833
+source: USACO Bronze 2018 Open
+title: Family Tree
+author: Danh Ta Chi Thanh
+---
+
+## Solution
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_family_bronze_open18.html)
+
+## Alternative Solution - BFS
+
+This problem can also be solved by using BFS to find the common ancestors and the distance between the two cows and their nearest common ancestor.
+

```

Comment:
Provide more explanation to your approach...

Reply:
Since you're using BFS (a gold topic), I recommend you copy this problem on the BFS module for gold and make the difficulty `very easy`

Reply:
Oops I don't think this is a good problem to put in gold BFS considering the intended solution isn't bfs :P